helm_deploy/values-preprod.yaml

```diff
@@ -6,6 +6,9 @@ generic-service:
 
   ingress:
     host: calculate-release-dates-api-preprod.hmpps.service.justice.gov.uk
+    annotations:
+      external-dns.alpha.kubernetes.io/set-identifier: calculate-release-dates-api-calculate-release-dates-api-preprod-green
+      external-dns.alpha.kubernetes.io/aws-weight: 100
 
```

Comment:
I think you can add these simply by specifying `contextColour: green` - the generic-service takes care of actually adding the annotations. Take a look at `create-and-vary-a-licence-api` as an example. Make sure to use the latest generic service version too.

Reply:
Thanks @timharrison-moj - have changed it

Reply:
Actually - sorry - you're still in live-1 cluster?  That means it should be contextColour = blue.

Reply:
No ingress for your API service in the live cluster :-( - I just checked.